src/build_multiversion.zig

```diff
@@ -786,16 +786,14 @@ fn self_check_enabled(target: Target) bool {
 
 fn self_check(shell: *Shell, multiversion: []const u8, past_releases: []const []const u8) !void {
     assert(past_releases.len > 0);
-    try shell.exec_options(
-        .{ .echo = false },
```

Comment:
I think there's no need to proactively echo things anymore, the command is printed iff it errors/hangs